### ISSUE TITLE
Batch norm for DDP

### ISSUE THREAD

**Issue by tky823** [tky823/Audyn#224]
## Summary
`nn.SyncBatchNorm` should be used for DDP instead of ``nn.BatchNormNd``.

**Comment by tky823** [comment]
addressed by #317
